### ISSUE TITLE
Use session authentication simply

### ISSUE THREAD

**Issue by the-red** [kintone/kintone-js-sdk#33]
The constructor of `Record, BulkRequest, App, File` objects requires `connection` argument.
When using session authentication, it's hassle to write as follows.

```js
const kintoneRecord = new Record(new Connection())
```

I'd like to exec session authentication as follows.
(Other than guest space)
```js
const kintoneRecord = new Record()
```
If `connection` argument is omitted, how about making it equivalent to `new Connection()`?

**Comment by phongnhat19** [comment]
@the-red We've fixed this in version 0.3.1

**Comment by the-red** [comment]
Thank You!!!
